.changeset/thirty-beans-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-frontend/application-components': patch
+'@commercetools-local/visual-testing-app': patch
+---
+
+introduce xl prop for FormDialog component
```

Comment:
```suggestion
Introduced a new available size value (`xl`) for the `FormDialog` component.
```

Reply:
Thanks! Resolved in ab800c3c8bbe0ad26da2a6bf4e47786a007f9c76